django/contrib/gis/db/models/functions.py

```diff
@@ -249,11 +249,11 @@ def as_postgresql(self, compiler, connection):
         elif geo_field.geodetic(connection):
             # Geometry fields with geodetic (lon/lat) coordinates need special distance functions
             if self.spheroid:
-                self.function = 'ST_Distance_Spheroid'  # More accurate, resource intensive
+                self.function = connection.ops.spatial_function_name('DistanceSpheroid')
```

Comment:
Flake8 complains about the end-of-line comment making the line too long.
I don't think that particular comment is very helpful anyway. This is not really the right place to learn the difference between `DistanceSphere` and `DistanceSpheroid`. But, as there maybe are people reading the code for which the comment will be helpful, I'll move it to a new line before the assignment.